Create ui-c8y-1021-56-5-auto-refresh-gets-stuck-when-realtime-is-also-added-to-global.md (#3150)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-56-5-auto-refresh-gets-stuck-when-realtime-is-also-added-to-global.md

---
date: ""

> title: Resolved an issue where the dashboard’s auto-refresh counter would sometimes freeze

Reviewer: ```suggestion
title: Dashboard auto-refresh counter no longer freezes in certain scenarios
```

> title: Auto refresh gets stuck when realtime is also added to global date context (#8280) [GRAFT][release/cd] (#8709)

Reviewer 2: The Claude AI suggested to use the following title:
```suggestion
title: Resolved an issue where the dashboard’s auto-refresh counter would sometimes freeze
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YdSEScrEC
    label: Cockpit
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-62595
version: 1021.56.5
---

> Auto refresh gets stuck when realtime is also added to global date context (#8280) [GRAFT][release/cd] (#8709)

Reviewer 2: ```suggestion
Fixed an issue where the auto-refresh counter would freeze when a dashboard contained at least two widgets with different refresh mechanisms - one using real-time dashboard contexts and another using auto-refresh dashboard contexts. This combination now works properly for previously failing scenarios.
```

> Fixed an issue where the auto-refresh counter would freeze when a dashboard contained at least two widgets with different refresh mechanisms - one using real-time dashboard contexts and another using auto-refresh dashboard contexts. This combination now works properly for previously failing scenarios.

Reviewer: ```suggestion
Previously, the auto-refresh counter freezed when a dashboard contained at least two widgets with different refresh mechanisms - one using real-time dashboard contexts and another using auto-refresh dashboard contexts. This combination now works properly for previously failing scenarios.
```
